projects/storefrontapp-e2e-cypress/cypress/helpers/site-theme.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export function interceptToAddThemeCompnent() {
```

Comment:
[MINOR]
Let's leave a code comment here, that this is only a temporary solution, until we deploy the proper sample data to the CI server. And then this code should be removed.
Please also attach in the code comment a relevant JIRA ticket's number - for removing this piece of mocking logic.